run.js

```diff
@@ -242,12 +243,47 @@ async function buildTemplates(preProcessor, postProcessor, selectedTemplate) {
                             type: m.getTemplateType()
                         }
                         templateIndex[template.getIdentifier()] = indexData;
-        
+
                         // call the post template processor
                         await postProcessor(templateIndex, templatePath, template);
                     }
                     else {
+                      if (process.env.UPDATE_DROPDOWNS) {
+                        const templateVersions = Object.keys(templateIndex).filter(
```

Comment:
Please sort the keys by semantic version number rather than alphabetically. Otherwise we get a confusing list, like this, 
![image](https://user-images.githubusercontent.com/7544022/66820803-40bfd980-ef39-11e9-8c83-09311bc0ad7a.png)